src/use-resize-observer.ts

```diff
@@ -1,6 +1,6 @@
+import { ResizeObserver } from "@juggle/resize-observer";
```

Comment:
Juggle's `ResizeObserver` is a ponyfill so preferably we'd use the one on the `window` object when it's available